models/_sector/nft/trades/MIGRATION.md

```diff
@@ -0,0 +1,156 @@
+# Sector: NFT
+
+We're migrating the NFT sector spells to a new structure which will make implementing and maintaining marketplace abstractions a lot easier and more robust.
+
+## `nft.trades`
+
+- partition_by: `[blockchain, block_date]` 
```

Comment:
may be worthwhile for us to do deeper analysis on the partition columns, for example:
- run group by these fields, grab counts, see if any combo have much too small volume of data for a partition (could be different on trino vs. spark)
- potentially leverage internal analytics tools to see usage on `nft.trades` to find most common group by/join conditions/etc that help partitions, though these fields are most likely candidates (@augustog can help us here)

Reply:
I agree we need to explore the possibilities here further.
For now I went with blockchain as most of the queries and dashboards I've seen are still blockchain specific so this setup should allow for easy/efficient filtering. The thought behind this is also that we don't want blockchains with a lot of (spam) NFT traffic to influence compute times for queries running only on mainnet for example.

Reply:
Task so we don't forget and circle back on this. 
https://dune.height.app/T-24057?task=T-27749